emission/tests/analysisTests/modeinferTests/TestPipeline.py

```diff
@@ -42,13 +48,11 @@ def setUp(self):
     for section in self.SectionsColl.find():
       section['section_start_datetime'] = self.dayago
       section['section_end_datetime'] = self.dayago + timedelta(hours = 1)
-      if (section['confirmed_mode'] == 5):
+      if (('confirmed_mode' in section and section['confirmed_mode'] == 5) or ('sensed_mode' in section and section['sensed_mode'] == 5)):
```

Comment:
`sensed_mode == 5` doesn't mean the same thing any more.
from `emission/core/wrapper/motionactivity.py`, `TILTING = 5`